### DISCUSSION TITLE
MPV 是否可以用AMD自带的补帧

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#340] [mpv 通用]
类似下面这种方式，耗资源很少，我amd集显也能跑
https://zhuanlan.zhihu.com/p/443737935
但是不喜欢用potplayer，用mpv-lazy用惯了
有办法在MPV上直接用吗

**Comment by hooke007** [comment]
同理
https://github.com/hooke007/MPV_lazy/discussions/53

**Reply by micro-mieruko** [reply]
AMD新出的那个AFMF补帧也不行吗

**Reply by hooke007** [reply]
不行
